app/models/application.rb

```diff
@@ -20,10 +20,13 @@ class Application < ApplicationRecord
 
   default_scope { order("name ASC") }
 
+  ENVIRONMENTS_ORDER = ["integration", "integration EKS", "staging", "staging EKS", "production", "production EKS"].freeze
+
   def latest_deploy_to_each_environment
     return @latest_deploy_to_each_environment unless @latest_deploy_to_each_environment.nil?
 
-    environments = deployments.select("DISTINCT environment").map(&:environment)
+    environments = deployments.select("DISTINCT environment").map(&:environment).sort_by { |e| ENVIRONMENTS_ORDER.index(e) || Float::INFINITY }
```

Comment:
This appears to be retrieving environment names dynamically, and then sorting as per `ENVIRONMENTS_ORDER`. But now that you have a hardcoded list of environments on line 23, you really don't need this line at all - you could just make this `environments = ["integration"... etc`.

Reply:
I think we have changed this since and moved the sorting into a private method, I'm just taking a pulse check on whether or not the other environments should show, thank you.